js/js-array-methods_for-each/js/index.js

```diff
@@ -19,4 +19,18 @@ const colors = [
   "#A5A5A5",
 ];
 
-colors.forEach();
+const colorBox = document.querySelector("color-box");
```

Comment:
You actually don't need this. :) And also, this doesn't do anything, I think because 1) you would have to write **document.querySelector('[data-js="color-box"]')** and 2) there is no HTML-element with the **attribute data-js=color-box**.